lib/dogwatch/dogfile.rb

```diff
@@ -18,11 +18,22 @@ def configure(dogfile, api_key, app_key, timeout)
     # @param [Proc] block
     def create(&block)
       monitor = instance_eval(IO.read(@dogfile), @dogfile, 1)
-      monitor.config = @config
-      monitor.client
 
-      monitor.get
-      monitor.responses.each { |r| block.call(r) }
+      if monitor.is_a?(DogWatch::Monitor)
+        monitor.config = @config
+        # monitor.client
```

Comment:
Was leaving these lines about `monitor.client`, `monitor.get`, and `monitor.responses` commented out intentional?